### DISCUSSION TITLE
Remove AI in MSFS ?

### DISCUSSION THREAD

**Discussion by fredogaming** [albar965/littlenavmap#905] [Q&A]
Hi, does anyone know how to remove just the AI at the airports ? I just landed in LEMD. And it was full with AI aircrafts on the ground, and the sim was stuttering just to update all of the aircrafts.

Thanks.

**Comment by albar965** [comment]
You might want to report this Asobo if MSFS is stuttering. 😉

In LNM you can simply disable AI display on the toolbar or menu `View`-> `Show AI and multiplayer aircraft` or disable transfer of AI in the connection dialog if it helps.

**Comment by fredogaming** [comment]
Yeah, i know how to ”show AI and multiplayer” but if i disable it the for example IVAO Aircrafts does not shows up. Skickades från E-post för Windows Från: Alexander BarthelSkickat: den 29 juli 2022 12:32Till: albar965/littlenavmapKopia: fredogaming; AuthorÄmne: Re: [albar965/littlenavmap] Remove AI in MSFS ? (Discussion #905) You might want to report this Asobo if MSFS is stuttering. 😉In LNM you can simply disable AI display on the toolbar or menu View-> Show AI and multiplayer aircraft or disable transfer of AI in the connection dialog if it helps.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by albar965** [comment]
> Yeah, i know how to ”show AI and multiplayer” but if i disable it the for example IVAO Aircrafts does not shows up.

I see. In next LNM you can disable AI and Online traffic separately. There is already an alpha version where you can do this: https://www.avsim.com/forums/topic/614851-little-navmap-alphadevelopment-2710-released-update-3/

Alex

**Comment by fredogaming** [comment]
Oh, very nice.Thanks alot mate  Skickades från E-post för Windows Från: Alexander BarthelSkickat: den 29 juli 2022 16:19Till: albar965/littlenavmapKopia: fredogaming; AuthorÄmne: Re: [albar965/littlenavmap] Remove AI in MSFS ? (Discussion #905) Yeah, i know how to ”show AI and multiplayer” but if i disable it the for example IVAO Aircrafts does not shows up.I see. In next LNM you can disable AI and Online traffic separately. There is already an alpha version where you can do this: https://www.avsim.com/forums/topic/614851-little-navmap-alphadevelopment-2710-released-update-3/Alex—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by fredogaming** [reply]
Hi again, i have tested the Dev release. It works...kinda.
If i tick the "online" i can see other online aircrafts in about 1 sec, then they are gone.
They shoes up again if i tick the AI button...

**Reply by albar965** [reply]
Sorry, will be fixed in the next alpha which should appear soonish.

**Reply by fredogaming** [reply]
Awesome, you don't have to say sorry mate....its an awesome freeware already. I have used it since it came out. :)
